### DISCUSSION TITLE
mobile: "upload images before videos" preference setting, or "upload smallest files first"

### DISCUSSION THREAD

**Discussion by sneak** [ente-io/ente#3228] [Enhancements]
When on shitty hotel wifi whilst traveling, it would be useful to me for the mobile uploader to prioritize uploading (relatively small) photos first, and upload the huge videos later, time and bandwidth permitting.   The preference setting might be better served as "upload order" with choices such as "smallest first, newest first, largest first, oldest first, random", where "smallest first" would probably achieve the same thing for my use case (where the majority of files are already uploaded).

Ideally I'd be able to choose the number of parallel upload slots, too (ideally with different settings for mobile data or wifi).

**Comment by sneak** [comment]
I guess for now I could disable "upload videos" (hadn't noticed that one), let it finish, then manually re-enable it (hopefully I will remember).

**Reply by ua741** [reply]
FYI: The mobile app currently uploads images before videos.

**Comment by sneak** [comment]
Oh, then nevermind. :)

**Comment by sneak** [comment]
There might be a bug in iOS 0.9.35, as it was stuck uploading some bigger (old) videos, but hadn't uploaded very recent photos.
